posydon/popsyn/binarypopulation.py

```diff
@@ -76,6 +76,8 @@
     'step_HMS_HMS', 'step_CO_HeMS', 'step_CO_HMS_RLO', 'step_CO_HeMS_RLO', 'step_detached','step_isolated','step_disrupted','step_initially_single', 'step_merged'
 ]
 
+POSYDON_METALLICITIES = [2e+00,1e+00,4.5e-01,2e-01,1e-01,1e-02,1e-03,1e-04]
```

Comment:
Those are the metallicities for v2. In future versions, we might get more. Hence, I'm not sure, whether we should hard code this here. Wouldn't it be better the infer a list from the used data set?

Reply:
I have removed it and passed the `metallicities` as an input from the `SyntheticPopulation` to the `BinaryPopulation`. If that is not used, then the `metallicity` is used for backwards compatibility.